packages/gatsby-image/src/index.js

```diff
@@ -295,13 +286,23 @@ class Image extends React.Component {
                 top: 0,
                 bottom: 0,
                 opacity: !this.state.imgLoaded ? 1 : 0,
-                transitionDelay: `0.35s`,
+                transitionDelay: initialDelay,
```

Comment:
this used to be `0.35s` now it's `0.25s` unsure if it matters (I don't think) I'll check it out.

Reply:
Check the commit history, there are explanations :) In this case, I noted that there was a discrepancy which was also causing a visual transition bug for me in a slider component. I assumed this was probably a typo and that the delays are meant to be in sync?

Reply:
The `transitionDelay` of `0.35s` was added by @KyleAMathews some time ago, via [this commit](https://github.com/gatsbyjs/gatsby/commit/76cda50d75dd9ea8950b74b8ebc97bc0380d2250) according to git blame. It was batched into the same commit as many other changes, so it's purpose isn't clear.

I think it's safe to share the `0.25s` value placeholder uses for transition delay?

Reply:
Investigating #12254 and working on a PR for that, I've since come to believe the `0,25s` value is never actually used now, but `bgcolor` should use `0,5s`. `initialDelay` loses it's purpose, but could be renamed to the duration/delay these components share(all for the shared fade-in transition).